Storage/Clients.swift

```diff
@@ -37,18 +37,18 @@ public struct RemoteClient: Equatable {
         self.device = json["device"].stringValue
```

Comment:
Ugh, all of these supposedly optional `.stringValue` lines — everything but name — should be `.string`. Fucking SwiftyJSON. @sleroux, do you agree?

These used to be `asString`, which was an optional.